### PR TITLE
changed loader to react.createclass

### PR DESCRIPTION
**Pull request by lukeschunk**
### Changed Loader to `React.createClass` syntax
* moved `getDefaultProps()` and `propTypes` inside of React.createClass
* changed it to `const Loader` 
* followed these guidelines for order: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-comp.md

### CODE REVIEW

File: src/components/Loader.js [hunk 1]
```diff
@@ -4,7 +4,26 @@ const Spin = require('./Spin');
 
 const StyleConstants = require('../constants/Style');
 
-class Loader extends React.Component {
+const Loader = React.createClass({
+
```
Comment (bsbeeks) [inline src/components/Loader.js]: Remove this line break

File: src/components/Loader.js [hunk 2]
```diff
@@ -69,21 +88,7 @@ class Loader extends React.Component {
...
 
-Loader.defaultProps = {
-  color: StyleConstants.Colors.PRIMARY,
-  isLoading: false,
-  isRelative: false,
-  isSmall: false,
-  children: 'LOADING...'
-};
 
 module.exports = Loader;
```
Comment (bsbeeks) [inline src/components/Loader.js]: Indentation is off with all your changes. We use 2 spaces for indentation.

Comment (lukeschunk) [inline src/components/Loader.js]: Sorry about that. I always have that issue for some reason even though I switched it in my atom. I usually just go back through on github and fix the changes...

**Comment by bsbeeks** [comment]
Make sure it passes Travis.

File: src/components/Loader.js (hunk 2 above)
Comment (bsbeeks) [inline src/components/Loader.js]: You should be able to set your default spacing in Atom to 2 spaces (no tabs).

**Comment by lukeschunk** [comment]
@bsbeeks I will :ok_hand:

**Pull request merged**